AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Support/Utils/AuthErrorHelper.swift

```diff
@@ -7,6 +7,10 @@
 
 import Amplify
 import AWSMobileClient
+import SafariServices
+#if canImport(AuthenticationServices)
+import AuthenticationServices
+#endif
```

Comment:
Please verify that we actually need the `canImport` guard for this. My recollection is that Apple frameworks are constructed with appropriate availability guards to avoid this. Test by importing a module with this code change in a project with an iOS 9 deployment target (using SFSafari style hosted UI), and one iOS 14 (using ASAuthenticationSession). If they both work as expected, then we shouldn't need this `canImport`